build/build-smokehouse-bundle.js

```diff
@@ -12,19 +12,25 @@ const {LH_ROOT} = require('../root.js');
 const distDir = `${LH_ROOT}/dist`;
 const bundleOutFile = `${distDir}/smokehouse-bundle.js`;
 const smokehouseLibFilename = './lighthouse-cli/test/smokehouse/frontends/lib.js';
-const smokehouseCliFilename =
-  require.resolve('../lighthouse-cli/test/smokehouse/lighthouse-runners/cli.js');
+const smokehouseCliFilename = `${LH_ROOT}/lighthouse-cli/test/smokehouse/lighthouse-runners/cli.js`;
 
 async function build() {
   const bundle = await rollup.rollup({
     input: smokehouseLibFilename,
     context: 'globalThis',
     plugins: [
-      rollupPlugins.nodeResolve(),
-      rollupPlugins.commonjs(),
       rollupPlugins.shim({
-        [smokehouseCliFilename]: 'export default {}',
+        [smokehouseCliFilename]:
+          'export function runLighthouse() { throw new Error("not supported"); }',
+      }),
+      // TODO(esmodules): brfs does not support es modules.
+      rollupPlugins.brfs({
```

Comment:
I'm gonna work on replacing `brfs` tomorrow.

Reply:
most of this appears to be brfs-related, does that mean this PR is obsolete tomorrow?

Reply:
> I'm gonna work on replacing `brfs` tomorrow.

wait, I literally just wrote a replacement in https://github.com/GoogleChrome/lighthouse/pull/12771#issuecomment-928372085. Do you want me to just open that up as a separate PR?

Reply:
yes please!

Reply:
actually, I'll take care of it